content/en/docs/reference/command-line-tools-reference/feature-gates/selinux-mount.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SELinuxMount
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Speeds up container startup by allowing kubelet to mount volumes
+for a Pod directly with the correct SELinux label instead of changing each file on the volumes
+recursively. It builds on top of `SELinuxMountReadWriteOncePod` feature gate and extends
+the implementation to all volumes.
+
+`SELinuxMount` feature gate requires feature gate `SELinuxMountReadWriteOncePod` to
```

Comment:
(nit)
```suggestion
Enabling the `SELinuxMount` feature gate requires the feature gate `SELinuxMountReadWriteOncePod` to
```

It's useful to document what happens if you don't do that (eg API server bails out during initialization).